pyproject.toml

```diff
@@ -7,3 +7,33 @@ target_version = ["py36"]
 
 [tool.setuptools_scm]
 local_scheme = "no-local-version"
+
+[tool.isort]
+profile = "black"
+known_third_party = ["humanize"]
```

Comment:
```suggestion
```

@hugovk removing this moves `humanize` to the first party section in the test imports. I think that's desirable because `humanize` is a first party library